migrate/20240122_pg_resource_fw_rules.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:postgres_resource_firewall_rule) do
+      column :id, :uuid, primary_key: true, null: false
+      column :ip, :cidr, null: false
```

Comment:
Should we call this cidr as well? IP gives the impression that this is single IP address, not a range.

Reply:
I have the same question.

Reply:
Makes sense, I have refactored the whole code to use cidr. Thanks